action.yml

```diff
@@ -53,20 +53,50 @@ inputs:
 runs:
   using: 'composite'
   steps:
+    - name: Check if review is requested
```

Comment:
🔍 Consider adding a comment explaining the purpose of checking if a review is requested for clarity.